frontend/packages/console-app/src/components/quick-starts/controller/QuickStartConclusion.tsx

```diff
@@ -22,20 +22,32 @@ const QuickStartConclusion: React.FC<QuickStartConclusionProps> = ({
   onQuickStartChange,
   onTaskSelect,
 }) => {
+  let failedTaskCount = 0;
   return (
     <>
-      {tasks.map((task, index) => (
-        <TaskHeader
-          key={task.title}
-          title={task.title}
-          taskIndex={index + 1}
-          size="md"
-          taskStatus={allTaskStatuses[index]}
-          onTaskSelect={onTaskSelect}
-        />
-      ))}
-      <SyncMarkdownView content={conclusion} />
-      {nextQuickStart && (
+      {tasks.map((task, index) => {
+        if (allTaskStatuses[index] === QuickStartTaskStatus.FAILED) {
```

Comment:
You don't need to check the status for each task like this. Just check if `allTaskStatuses` has at least one `failed` status in the array. If yes then show error message else show conclusion.

Reply:
I thought i would avoid introducing a new loop and use the existing one

Reply:
updated the check to `const hasFailedTask = allTaskStatuses.some((t) => t === QuickStartTaskStatus.FAILED);`